target/linux/ramips/dts/mt7621_netgear_r6260_r6350_r6850.dtsi

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "mediatek,mt7621-soc";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			gpios = <&gpio0 18 GPIO_ACTIVE_LOW>;
+		};
+
+		led_usb: usb {
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&xhci_ehci_port1>, <&ehci_port2>;
+			linux,default-trigger = "usbport";
+		};
+
+		led_internet: internet {
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wifi: wifi {
+			gpios = <&gpio0 16 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
```

Comment:
MediaTek MT7621 should support interrupt-driven gpio-button support.  Can you verify this works on your board?

Reply:
works, changed.